### ISSUE TITLE
create jetpack has failed to load correctly

### ISSUE THREAD

**Issue by Boogelbie** [PssbleTrngle/CreateJetpack#19]
### What happened?

create jetpack has failed to load correctly
(create, Flywheel and kotlin for forge were installed)
java.lang.reflect.InvocationTargetException: null

### Mod Version

1.2.1

### Minecraft Version

1.18

### Forge Version

Forge-40.1.73

### Relevant log output

```shell
---- Minecraft Crash Report ----
// Don't do that.

Time: 8/16/22, 5:24 PM
Description: Mod loading error has occurred

java.lang.Exception: Mod Loading has failed
	at net.minecraftforge.logging.CrashReportExtender.dumpModLoadingCrashReport(CrashReportExtender.java:55) ~[forge-1.18.2-40.1.73-universal.jar%23122!/:?] {re:classloading}
	at net.minecraftforge.client.loading.ClientModLoader.completeModLoading(ClientModLoader.java:170) ~[forge-1.18.2-40.1.73-universal.jar%23122!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.lambda$new$1(Minecraft.java:556) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:blueprint.mixins.json:client.MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.Util.m_137521_(Util.java:397) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:classloading,re:mixin}
	at net.minecraft.client.Minecraft.lambda$new$2(Minecraft.java:550) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:blueprint.mixins.json:client.MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.LoadingOverlay.m_6305_(LoadingOverlay.java:135) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:879) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1045) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:blueprint.mixins.json:client.MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:664) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:blueprint.mixins.json:client.MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%23117!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.73.jar%2317!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.possible_triangle.create_jetpack.Content.<clinit>(Content.kt:61) ~[create_jetpack-1.2.1.jar%2376!/:1.2.1] {re:classloading}
-- MOD create_jetpack --
Details:
	Caused by 0: java.lang.reflect.InvocationTargetException
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
		at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
		at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
		at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.73.jar%23119!/:?] {}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.73.jar%23118!/:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
		at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {}
		at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

	Mod File: /C:/Users/Basti/curseforge/minecraft/Instances/Create Updated/mods/create_jetpack-1.2.1.jar
	Failure message: Create Jetpack (create_jetpack) has failed to load correctly
		java.lang.reflect.InvocationTargetException: null
	Mod Version: 1.2.1
	Mod Issue URL: NOT PROVIDED
	Exception message: java.lang.NoSuchMethodError: 'com.simibubi.create.repack.registrate.util.nullness.NonNullSupplier com.simibubi.create.foundation.data.CreateRegistrate.lazy(java.lang.String)'
Stacktrace:
	at com.possible_triangle.create_jetpack.Content.<clinit>(Content.kt:61) ~[create_jetpack-1.2.1.jar%2376!/:1.2.1] {re:classloading}
	at com.possible_triangle.create_jetpack.CreateJetpackMod.<init>(CreateJetpackMod.java:14) ~[create_jetpack-1.2.1.jar%2376!/:1.2.1] {re:classloading}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.73.jar%23119!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.73.jar%23118!/:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}


-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 3600047104 bytes (3433 MiB) / 4882169856 bytes (4656 MiB) up to 10636754944 bytes (10144 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i5-12400F
	Identifier: Intel64 Family 6 Model 151 Stepping 5
	Microarchitecture: unknown
	Frequency (GHz): 2.50
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1050 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1c82
	Graphics card #0 versionInfo: DriverVersion=31.0.15.1659
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 30021.42
	Virtual memory used (MB): 15048.32
	Swap memory total (MB): 13824.00
	Swap memory used (MB): 104.24
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx10144m -Xms256m
	ModLauncher: 9.1.3+9.1.3+main.9b69c82a
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		 mixin PLUGINSERVICE 
		 eventbus PLUGINSERVICE 
		 slf4jfixer PLUGINSERVICE 
		 object_holder_definalize PLUGINSERVICE 
		 runtime_enum_extender PLUGINSERVICE 
		 capability_token_subclass PLUGINSERVICE 
		 accesstransformer PLUGINSERVICE 
		 runtimedistcleaner PLUGINSERVICE 
		 mixin TRANSFORMATIONSERVICE 
		 fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		kotlinforforge@3.7.1
		javafml@null
	Mod List: 
		create-stuff-additions1.18.2_v1.1.8.jar           |Create Stuff Additions        |create_stuff_additions        |1.1.8               |COMMON_SET|Manifest: NOSIGNATURE
		bigbrain-1.5.3.jar                                |Big Brain                     |bigbrain                      |1.5.3               |COMMON_SET|Manifest: NOSIGNATURE
		TheComfortZone-1.18.2-2.0.0.jar                   |The Comfort Zone              |thecomfortzone                |1.18.2-2.0.0        |COMMON_SET|Manifest: NOSIGNATURE
		jei-1.18.2-9.7.1.232.jar                          |Just Enough Items             |jei                           |9.7.1.232           |COMMON_SET|Manifest: NOSIGNATURE
		Galosphere-1.18.2-1.0.4.jar                       |Galosphere                    |galosphere                    |1.18.2-1.0.4        |COMMON_SET|Manifest: NOSIGNATURE
		goblintraders-1.7.2-1.18.2.jar                    |Goblin Traders                |goblintraders                 |1.7.2               |COMMON_SET|Manifest: NOSIGNATURE
		caelus-forge-1.18.1-3.0.0.2.jar                   |Caelus API                    |caelus                        |1.18.1-3.0.0.2      |COMMON_SET|Manifest: NOSIGNATURE
		stalwart-dungeons-1.18.2-1.2.2.jar                |Stalwart Dungeons             |stalwart_dungeons             |1.2.2               |COMMON_SET|Manifest: NOSIGNATURE
		create_jetpack-1.2.1.jar                          |Create Jetpack                |create_jetpack                |1.2.1               |ERROR     |Manifest: NOSIGNATURE
		XaerosWorldMap_1.26.2_Forge_1.18.2.jar            |Xaero's World Map             |xaeroworldmap                 |1.26.2              |COMMON_SET|Manifest: NOSIGNATURE
		shutupexperimentalsettings-1.0.5.jar              |Shutup Experimental Settings! |shutupexperimentalsettings    |1.0.5               |COMMON_SET|Manifest: NOSIGNATURE
		Decorative Blocks-forge-1.18.2-2.1.1.jar          |Decorative Blocks             |decorative_blocks             |2.1.1               |COMMON_SET|Manifest: NOSIGNATURE
		feature_nbt_deadlock_be_gone_forge-2.0.0+1.18.2.ja|Feature NBT Deadlock Be Gone  |feature_nbt_deadlock_be_gone  |2.0.0+1.18.2        |COMMON_SET|Manifest: NOSIGNATURE
		create-confectionery1.18.2_v1.0.8.jar             |Create Confectionery          |create_confectionery          |1.0.8               |COMMON_SET|Manifest: NOSIGNATURE
		simpleplanes-1.18.2-4.13.4.jar                    |Simple Planes                 |simpleplanes                  |1.18.2-4.13.4       |COMMON_SET|Manifest: NOSIGNATURE
		blueprint-1.18.2-5.4.3.jar                        |Blueprint                     |blueprint                     |5.4.3               |COMMON_SET|Manifest: NOSIGNATURE
		boatload-1.18.2-3.0.0.jar                         |Boatload                      |boatload                      |3.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		buzzier_bees-1.18.2-4.0.0.jar                     |Buzzier Bees                  |buzzier_bees                  |4.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		savage_and_ravage-1.18.2-4.0.0.jar                |Savage & Ravage               |savage_and_ravage             |4.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		allurement-1.18.2-2.0.0.jar                       |Allurement                    |allurement                    |2.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		wandering-bag-1.18.2-1.0.3.jar                    |Wandering Bag                 |wandering_bag                 |1.0.4               |COMMON_SET|Manifest: NOSIGNATURE
		cloth-config-6.2.62-forge.jar                     |Cloth Config v4 API           |cloth_config                  |6.2.62              |COMMON_SET|Manifest: NOSIGNATURE
		soundphysics-forge-1.18.2-1.0.6.jar               |Sound Physics Remastered      |sound_physics_remastered      |1.18.2-1.0.6        |COMMON_SET|Manifest: NOSIGNATURE
		forge-1.18.2-40.1.73-universal.jar                |Forge                         |forge                         |40.1.73             |COMMON_SET|Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		neapolitan-1.18.2-3.0.0.jar                       |Neapolitan                    |neapolitan                    |3.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		idas_forge-1.3.3+1.18.2.jar                       |Integrated Dungeons and Struct|idas                          |1.3.3+1.18.2        |COMMON_SET|Manifest: NOSIGNATURE
		selene-1.18.2-1.17.9.jar                          |Selene                        |selene                        |1.18.2-1.17.9       |COMMON_SET|Manifest: NOSIGNATURE
		supplementaries-1.18.2-1.4.8.jar                  |Supplementaries               |supplementaries               |1.18.2-1.4.8        |COMMON_SET|Manifest: NOSIGNATURE
		corpse-1.18.2-1.0.1.jar                           |Corpse                        |corpse                        |1.18.2-1.0.1        |COMMON_SET|Manifest: NOSIGNATURE
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |COMMON_SET|Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		snowyspirit-1.18.2-1.3.2.jar                      |Snowy Spirit                  |snowyspirit                   |1.18.2-1.3.2        |COMMON_SET|Manifest: NOSIGNATURE
		another_furniture-forge-1.1.3-1.18.2.jar          |Another Furniture             |another_furniture             |1.1.3-1.18.2        |COMMON_SET|Manifest: NOSIGNATURE
		personality-1.18.2-2.0.0.jar                      |Personality                   |personality                   |2.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		CNB-1.18.2-1.3.2.jar                              |Creatures and Beasts          |cnb                           |1.3.2               |COMMON_SET|Manifest: NOSIGNATURE
		flywheel-forge-1.18.2-0.6.5.jar                   |Flywheel                      |flywheel                      |0.6.5               |COMMON_SET|Manifest: NOSIGNATURE
		create-1.18.2-0.5.0.d.jar                         |Create                        |create                        |0.5.0.d             |COMMON_SET|Manifest: NOSIGNATURE
		createplus-1.18.2_v0.5.1.jar                      |Create Plus                   |createplus                    |1.18.2_v0.5.1       |COMMON_SET|Manifest: NOSIGNATURE
		curios-forge-1.18.2-5.0.7.1.jar                   |Curios API                    |curios                        |1.18.2-5.0.7.1      |COMMON_SET|Manifest: NOSIGNATURE
		Patchouli-1.18.2-71.1.jar                         |Patchouli                     |patchouli                     |1.18.2-71.1         |COMMON_SET|Manifest: NOSIGNATURE
		Xaeros_Minimap_22.13.0_Forge_1.18.2.jar           |Xaero's Minimap               |xaerominimap                  |22.13.0             |COMMON_SET|Manifest: NOSIGNATURE
		create-factory-1.18.2-v1.6.jar                    |Create: Factory               |create_factory                |1.6                 |COMMON_SET|Manifest: NOSIGNATURE
		collective-1.18.2-4.37.jar                        |Collective                    |collective                    |4.37                |COMMON_SET|Manifest: NOSIGNATURE
		AutoRegLib-1.7-53.jar                             |AutoRegLib                    |autoreglib                    |1.7-53              |COMMON_SET|Manifest: NOSIGNATURE
		Quark-3.2-358.jar                                 |Quark                         |quark                         |3.2-358             |COMMON_SET|Manifest: NOSIGNATURE
		StorageDrawers-1.18.2-10.2.1.jar                  |Storage Drawers               |storagedrawers                |10.2.1              |COMMON_SET|Manifest: NOSIGNATURE
		littlelogistics-mc1.18.2-v1.2.6.jar               |Little Logistics              |littlelogistics               |1.2.6               |COMMON_SET|Manifest: NOSIGNATURE
		architectury-4.9.83-forge.jar                     |Architectury                  |architectury                  |4.9.83              |COMMON_SET|Manifest: NOSIGNATURE
		createchunkloading-1.2.1-forge.jar                |Create Chunkloading           |createchunkloading            |1.2.1               |COMMON_SET|Manifest: NOSIGNATURE
		appleskin-forge-mc1.18.2-2.4.1.jar                |AppleSkin                     |appleskin                     |2.4.1+mc1.18.2      |COMMON_SET|Manifest: NOSIGNATURE
		alexsdelight-1.18.2-1.2.3.jar                     |Alex's Delight                |alexsdelight                  |1.2.3               |COMMON_SET|Manifest: NOSIGNATURE
		citadel-1.11.3-1.18.2.jar                         |Citadel                       |citadel                       |1.11.3              |COMMON_SET|Manifest: NOSIGNATURE
		alexsmobs-1.18.6.jar                              |Alex's Mobs                   |alexsmobs                     |1.18.6              |COMMON_SET|Manifest: NOSIGNATURE
		FarmersDelight-1.18.2-1.1.2.jar                   |Farmer's Delight              |farmersdelight                |1.18.2-1.1.2        |COMMON_SET|Manifest: NOSIGNATURE
		NethersDelight-1.18.2-2.2.0.jar                   |Nethers Delight               |nethersdelight                |2.2                 |COMMON_SET|Manifest: NOSIGNATURE
		FarmersRespite-1.18.2-1.3.0.jar                   |Farmer's Respite              |farmersrespite                |1.18.1-1.3.0        |COMMON_SET|Manifest: NOSIGNATURE
		abnormals_delight-1.18.2-3.0.1.jar                |Abnormals Delight             |abnormals_delight             |3.0.1               |COMMON_SET|Manifest: NOSIGNATURE
		ferritecore-4.2.1-forge.jar                       |Ferrite Core                  |ferritecore                   |0.0NONE             |COMMON_SET|Manifest: 41:ce:50:66:d1:a0:05:ce:a1:0e:02:85:9b:46:64:e0:bf:2e:cf:60:30:9a:fe:0c:27:e0:63:66:9a:84:ce:8a
		curiouselytra-forge-1.18.1-5.0.1.0.jar            |Curious Elytra                |curiouselytra                 |1.18.1-5.0.1.0      |COMMON_SET|Manifest: NOSIGNATURE
		The_Undergarden-1.18.2-0.7.1.jar                  |The Undergarden               |undergarden                   |0.7.1               |COMMON_SET|Manifest: NOSIGNATURE
		create-chromaticreturn1.18.2_v1.1.jar             |Create: Chromatic Return      |createchromaticreturn         |1.0.0               |COMMON_SET|Manifest: NOSIGNATURE
		OverpoweredMending-1.18.2-2.4.1.jar               |OverpoweredMending            |overpoweredmending            |2.4.1               |COMMON_SET|Manifest: 3c:8e:df:6c:df:a6:2a:9f:af:64:ea:04:9a:cf:65:92:3b:54:93:0e:96:50:b4:52:e1:13:42:18:2b:ae:40:29
		geckolib-forge-1.18-3.0.34.jar                    |GeckoLib                      |geckolib3                     |3.0.34              |COMMON_SET|Manifest: NOSIGNATURE
		naturalist-forge-1.1.1-1.18.2.jar                 |Naturalist                    |naturalist                    |1.1.1               |COMMON_SET|Manifest: NOSIGNATURE
		domesticationinnovation-1.4.2-1.18.2.jar          |Domestication Innovation      |domesticationinnovation       |1.4.2               |COMMON_SET|Manifest: NOSIGNATURE
		druidcraftrg-0.0.12.jar                           |Druidcraft Regrown            |druidcraftrg                  |0.0.12              |COMMON_SET|Manifest: NOSIGNATURE
	Crash Report UUID: 192ca1d2-c3cf-4423-956e-005400d34952
	Flywheel Backend: GL33 Instanced Arrays
	FML: 40.1
	Forge: net.minecraftforge:40.1.73
```

**Comment by Boogelbie** [comment]
It said:
java.lang.reflect.InvocationTargetException: null

**Comment by PssbleTrngle** [comment]
There has been a new create release 2 days ago, that's probably what caused this.
Going to update the mod as soon as I get to it

**Comment by PssbleTrngle** [comment]
Should be fixed in version 1.2.2

**Comment by Boogelbie** [comment]
super thank you This is really an amazing mod and it fits so well to the Create feel
